app/graphql/users/validation.js

```diff
@@ -0,0 +1,22 @@
+const yup = require('yup'),
+  { validationError } = require('../../errors'),
+  alphanumericRegex = /^[0-9a-zA-Z]+$/,
```

Comment:
you could save this regular expressions on a constants file since you dont know if you will use it in other validations further along the path